src/components/CommentInfo/CommentInfo.tsx

```diff
@@ -1,5 +1,35 @@
 import React from 'react';
+import { Comment } from '../../types/Comment';
 
-export const CommentInfo: React.FC = () => (
-  <>Put the comment here</>
-);
+type Props = {
+  comment: Comment;
+};
+
+export const CommentInfo: React.FC<Props> = ({ comment }) => {
+  const {
+    name,
+    email,
+    body,
+  } = comment;
+
+  return (
+    <div className="CommentInfo">
+      <div className="CommentInfo__title">
+        <strong className="CommentInfo__name">{name}</strong>
+
```

Comment:
Adding a space at the beginning of comments to make them easier to read. Use the `spaced-comment` rule in ESLint 📖